### PR TITLE
MSI Z690-A v1.1.4 & MSI Z790-P v0.9.2

### PR DESCRIPTION
**Pull request by SergiiDmytruk**

### CODE REVIEW

File: configs/config.msi_ms7d25_ddr4 [hunk 1]
```diff
@@ -25,6 +25,9 @@ CONFIG_PCIEXP_SUPPORT_RESIZABLE_BARS=y
 CONFIG_PCIEXP_LANE_ERR_STAT_CLEAR=y
 CONFIG_DRIVERS_EFI_VARIABLE_STORE=y
 CONFIG_DRIVERS_EFI_FW_INFO=y
+CONFIG_DRIVERS_EFI_MAIN_FW_GUID="0f3e8ba6-19d3-461b-bbbc-8b5689fe8097"
```
Comment (krystian-hebel) [inline configs/config.msi_ms7d25_ddr4:28]: Should we consider adding a check to CI that tests whether all `MAIN_FW_GUID`s are unique?

Comment (krystian-hebel) [reply configs/config.msi_ms7d25_ddr4:28]: And I guess `MAIN_FW_VERSION` could also be checked against `LOCALVERSION`.

File: .github/scripts/capsule-lint.sh [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    local rc=${hex:8:2}
+
+    local ver="v$((0x$major)).$((0x$minor)).$((0x$bugfix))"
+    if [ "$((0x$rc))" -lt 128 ]; then
+        ver="$ver-rc$((rc))"
+    fi
+    echo "$ver"
+}
+
+if [ -d .configs ]; then
```
Comment (krystian-hebel) [inline .github/scripts/capsule-lint.sh:30]:
```suggestion
if [ ! -d configs ]; then
```

**Review by krystian-hebel — COMMENTED** [review]
One small typo in the script, otherwise looks good. I also learned a few bash tricks here :slightly_smiling_face:

File: .github/scripts/capsule-lint.sh (hunk 2 above)
Comment (SergiiDmytruk) [reply .github/scripts/capsule-lint.sh]: Nice, complementary typos.  https://github.com/Dasharo/coreboot/pull/565/commits/b18c294e64a2c77e95ec6fd6089d46067e617181

**Review by krystian-hebel — APPROVED** [review]

**Pull request merged**